SECP-service/pom.xml

```diff
@@ -56,7 +56,16 @@
             <artifactId>websocket-server</artifactId>
             <version>9.4.7.v20170914</version>
         </dependency>
-
+        <dependency>
+            <groupId>commons-validator</groupId>
+            <artifactId>commons-validator</artifactId>
+            <version>1.4.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.kenai.nbpwr</groupId>
+            <artifactId>edu-umd-cs-findbugs-annotations</artifactId>
```

Comment:
What's the use of this new dependency?? We already have a maven findbug dependency.